vspd.go

```diff
@@ -51,6 +51,60 @@ func (mw *MultiWallet) NewVSPD(baseURL string, walletID int, sourceAccountNumber
 	}
 }
 
+func (mw *MultiWallet) SaveVSPConfigValue(url string) (error, *GetVspInfoResponse) {
+	v := &VSPD{baseURL: url, httpClient: new(http.Client)}
+	info, err := v.GetInfo()
```

Comment:
Why do we need to save vsp host config? That sounds like data that should be fetched on demand because the values could change anytime.